test/cors.spec.js

```diff
@@ -0,0 +1,20 @@
+let helper = require('./helper');
+
+describe('CORS', function () {
+  it('GET /stories should allow cross origin', function(done) {
+    helper.request()
+          .get('/stories')
+          .end(function(err, res) {
+            res.should.to.have.header('Access-Control-Allow-Origin');
```

Comment:
yeah, I think checking for * in the first instance is good - we can restrict later - I guess I was just thinking that the test is somehow incomplete if we only check the header key and not the value, what do you think?